core/src/datayoga_core/blocks/relational/write/block.py

```diff
@@ -28,9 +50,9 @@ def init(self, context: Optional[Context] = None):
         logger.debug(f"Initializing {self.get_block_name()}")
 
         connection = utils.get_connection_details(self.properties.get("connection"), context)
-        db_type = connection.get("type")
+        db_type = connection.get("type").lower()
         engine_url = sa.engine.URL.create(
-            drivername=db_type,
+            drivername=get_driver_name(db_type),
```

Comment:
this should just be a default. we should support passing the driver name explicitly as well